lib/puppet/functions/eyaml_lookup_key.rb

```diff
@@ -0,0 +1,73 @@
+# @since 4.9.0
+#
+Puppet::Functions.create_function(:eyaml_lookup_key) do
+  unless Puppet.features.hiera_eyaml?
+    raise Puppet::DataBinding::LookupError, 'Lookup using eyaml lookup_key function is not supported without hiera_eyaml? library'
```

Comment:
"not supported without" is a double negation. There is also a '?' in the error message. Suggest rephrasing to something like "is only supported when the hiera_eyaml library is present"